homestar-runtime/src/settings.rs

```diff
@@ -98,6 +98,9 @@ pub struct Network {
     pub(crate) workflow_quorum: usize,
     /// Pubkey setup configuration
     pub(crate) keypair_config: PubkeyConfig,
+    /// Multiaddrs of the bootstrap nodes to connect to on startup
+    #[serde_as(as = "Vec<serde_with::DisplayFromStr>")]
+    pub(crate) bootstrap_addresses: Vec<libp2p::Multiaddr>,
```

Comment:
maybe `bootstrap_peers`?  Addresses comes through on the type.

Reply:
as this is the name used as the key for the config file i would like to stay a bit explicit on the name.